packages/third-parties/src/google/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * This is an autogenerated file by update-third-parties.js
+ */
+import React from 'react'
+import Script from 'next/script'
+
+import ThirdPartyScriptEmbed from '../ThirdPartyScriptEmbed'
+import * as Types from '../types/google'
+
+// Embed a Google Maps embed on your webpage
+export function GoogleMapsEmbed(args: Types.GoogleMapsEmbed) {
+  return (
+    <ThirdPartyScriptEmbed
+      height={args.height || null}
+      width={args.width || null}
+      content={`<iframe loading="lazy" src="https://www.google.com/maps/embed/v1/${args.mapMode}?key=${args.apiKey}&${args.parameters}" width=${args.width} height=${args.height} style=${args.style} loading=${args.loading} allowfullscreen=${args.allowfullscreen} referrerpolicy="no-referrer-when-downgrade"></iframe>`}
```

Comment:
Looks like there are duplicate `loading` attributes defined in the HTML here - one is "lazy" and one comes from "args.loading". I imagine we should combine the logic into one attribute so we fallback to the "lazy" if there isn't an override value?

Reply:
@janicklas-ralph Yeah this shouldn't be an issue anymore once you've upgraded to [1.0.19](https://github.com/GoogleChromeLabs/third-party-capital/blob/main/src/third-parties/google-maps-embed/data.json#L8). I've separated out all the attributes instead of having everything in one single `content` property which should make adding overrides easier.